packages/manfred2md/src/engine/md-parts/studies-section/studies-section.ejs

```diff
@@ -21,12 +21,12 @@
 - **Fecha de inicio y finalización:** <%= study.startDate %> - <%= study.finishDate === "" ? "Actualidad" : study.finishDate %>
     <%_ } -%>
 
-    <%_ if ( study?.institution ) { -%>
+    <%_ if ( study?.institution.name ) { -%>
 - **Nombre de Institución:** <%= study.institution.name %>
     <%_ } -%>
 
     <%_ if ( study.institution.location?.region || study.institution.location?.country ) { -%>
-- **Ubicación de la Institución:** <%= study.institution.location.region %>, <%= study.institution.location.country %>
+- **Ubicación de la Institución:** <%= study.institution.location.region ? ( study.institution.location?.country ? study.institution.location.region + ", " : study.institution.location.region) : "" %><%= study.institution.location.country %>
```

Comment:
It is not correct. If the region is informed but not the country, the country will always be displayed. This is not a problem because the mapper returns an empty string, but you are controlling country info conditionally. The correct form is:
`- **Ubicación de la Institución:** <%= study.institution.location.region ? ( study.institution.location?.country ? study.institution.location.region + ", " + study.institution.location.country : study.institution.location.region) : "" %><%= study.institution.location.country %>`